doc/nrf/libraries/debug/cpu_load.rst

```diff
@@ -37,12 +37,6 @@ Usage
 
```

Comment:
The Configuration section is missing information about how to enable the library. It is required by the [template guidelines](https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/templates/library_template_README.html). Please add it.

The current contents of Configuration are missing links to Kconfig options -- please add them using `:kconfig:option:` formatting.

Reply:
added